AEPMessaging/Sources/schemas/HtmlContentSchemaData.swift

```diff
@@ -16,7 +16,10 @@ import Foundation
 @objc(AEPHtmlContentSchemaData)
 @objcMembers
 public class HtmlContentSchemaData: NSObject, Codable {
+    /// Represents the content of the HtmlContentSchemaData object.  Its value's type is determined by `format`.
     public let content: String
+
```

Comment:
nit: `For now, this value is always "text/html"`

Reply:
i think i'll use `.textHtml` here - since i'm representing a `ContentType` enum value.

wdyt?